lib/core/src/lib/viewer/components/viewer.component.ts

```diff
@@ -559,4 +559,16 @@ export class ViewerComponent<T> implements OnDestroy, OnInit, OnChanges {
         this.urlFile = '';
         this.blobFile = null;
     }
+
+    get shouldShowDivider(): boolean {
+        if (!this.showToolbarDividers) {
+            return false;
+        }
+        const hasLeftSidebar = this.allowLeftSidebar;
```

Comment:
Why do you check left sidebar here and left close button position? Those are not the case for this divider as it is displayed after the title anyway, the proper conditions to display it or not would be `menu open with` or `adf-viewer-toolbar-actions` if one of the is present this divider should be shown